images/linux/scripts/installers/configure-environment.sh

```diff
@@ -25,3 +25,7 @@ chmod -R 777 $AGENT_TOOLSDIRECTORY
 # https://www.elastic.co/guide/en/elasticsearch/reference/current/vm-max-map-count.html
 # https://www.suse.com/support/kb/doc/?id=000016692
 echo 'vm.max_map_count=262144' | tee -a /etc/sysctl.conf
+
+# Create symlink for tests running
+chmod -R 777 $HELPER_SCRIPTS/invoke-tests.sh
```

Comment:
IMHO, `chmod +x` should be enough.
Also we don't need `-R` since we apply it to the single file.

Reply:
Fixed, thanks!